src/app/features/room-nav/RoomNavItem.tsx

```diff
@@ -217,9 +224,21 @@ export function RoomNavItem({
   const { focusWithinProps } = useFocusWithin({ onFocusWithinChange: setHover });
   const [menuAnchor, setMenuAnchor] = useState<RectCords>();
   const unread = useRoomUnread(room.roomId, roomToUnreadAtom);
+  const {
+    activeCallRoomId,
+    setActiveCallRoomId,
+    setViewedCallRoomId,
+    isChatOpen,
+    toggleChat,
+    hangUp,
+  } = useCallState();
   const typingMember = useRoomTypingMember(room.roomId).filter(
     (receipt) => receipt.userId !== mx.getUserId()
   );
+  const { navigateRoom } = useRoomNavigate();
```

Comment:
Add: Indicate active call
```suggestion
  const isActiveCall = isCallActive && activeCallRoomId === room.roomId;
  const { navigateRoom } = useRoomNavigate();
```